docs/growth/devsecops/README.md

```diff
@@ -15,3 +15,4 @@ This section will cover advanced DevSecOps topics like DevSecOps Transformation
 
 - [Ultimate DevSecOps library](https://github.com/sottlmarek/DevSecOps): A library contains list of tools and methodologies accompanied with resources.
 - [Awesome DevSecOps](https://github.com/devsecops/awesome-devsecops): An authoritative list of awesome DevSecOps tools with the help from community experiments and contributions.
+- [DevSecOpUniversity](https://www.practical-devsecops.com/devsecops-university/): comprehensive collection of DevSecOps Learning Resources like Books, Tutorials, Infographics, Tools and much more.
```

Comment:
`DevSecOpUniversity` => `DevSecOps University`

`comprehensive collection of DevSecOps Learning Resources like Books, Tutorials, Infographics, Tools and much more.` => `A comprehensive collection of DevSecOps learning resources like books, tutorials, infographics, tools, and much more.`